Source/JavaScriptCore/PlatformJSCOnly.cmake

```diff
@@ -1,3 +1,11 @@
+if (ENABLE_JSC_GLIB_API)
```

Comment:
I don't know much about cmake, but don't you need to define this option in OptionsJSCOnly.cmake ? We should also make sure that this options depends on enabling the glib event loop